src/rules/hi.toml

```diff
@@ -0,0 +1,21 @@
+min_trimmed_length = 3
+min_word_count = 5
```

Comment:
Can you elaborate the change from 1 to 5 here? How do smaller sentences below 5 words look like? Would there be some that would be valid?

Reply:
Having a lower limit will result in lot of sentences smaller than 5 words, which will look like:
यह बड़ा है (this is big)
और कहाँ है (where else is this)
क्या मिलेगा (what will you get)
..and so on

These don't seem to be very useful and will dominate the resulting dataset. What do you think?

Reply:
Smaller sentences are also OK if they make sense. How many are we losing when this is applied, what's the total with and without these sentences?